README.md

```diff
@@ -437,7 +468,58 @@ Just add the HTML element in your code, and you're all set.
 <radix-connect-button />
 ```
 
-Currently you as the developer have no control over the styling. A complete make-over is coming shortly with more customization options to fit your dApp's branding needs.
+## Styling
+
+Configure the √ Connect Button to fit your dApp's branding.
+
+### Themes
+
+Available themes:
+
+- `radix-blue` (default)
+- `black`
+- `white-with-outline`
+- `white`
+
+```typescript
+rdt.buttonApi.setTheme('black')
+```
+
+### Modes
+
+Available modes:
+
+- `light` (default)
+- `dark`
+
+```typescript
+rdt.buttonApi.setMode('dark')
+```
+
+### CSS variables
+
+There are three CSS variables available:
+
+- `--radix-connect-button-width` (default 138px)
+- `--radix-connect-button-height` (default 42px)
+- `--radix-connect-button-border-radius` (default 0px)
+
+```css
+body {
+  --radix-connect-button-width: 200px;
+  --radix-connect-button-height: 42px;
+  --radix-connect-button-border-radius: 12px;
+}
+```
+
+### Compact mode
+
+Setting `--radix-connect-button-width` below `138px` will enable compact mode.
+
+### Sandbox
+
+Play around with the different configurations on the
+[sandbox environment](https://connect-button-storybook.radixdlt.com/)
```

Comment:
IMO this should go to CB `README.md` and be linked from here

Reply:
We should probably keep it in both places for better readability.